package.json

```diff
@@ -7,7 +7,9 @@
     "watch": "lerna run watch",
     "test": "lerna run test",
     "lint": "lerna run lint",
-    "lint:fix": "lerna run lint:fix"
+    "lint:fix": "lerna run lint:fix",
+    "prebuild": "npx ts-node scripts/generate-schemas.ts",
+    "pretest": "npx ts-node scripts/generate-schemas.ts"
```

Comment:
`prebuild` and `pretest` are the same,  are they suppose to be different or is there a reason for having both?

Reply:
These are `npm` `pre-` hooks. So you can create a `prefoo` that runs before `foo`. Theoretically, it may be possible to do:

```
"generate-schema": "npx ts-node scripts/generate-schemas.ts",
"prebuild": "npm run generate-schema",
"pretest": npm run generate-schema",
```

to somewhat de-duplicate. https://www.yld.io/blog/using-npm-pre-and-post-hooks/

Reply:
Yep.  I was just happy to get it to work.
Would `precdk` be similar?